### ISSUE TITLE
Enter does create a new line in text editor

### ISSUE THREAD

**Issue by gabriel-joy** [nextcloud/ios#1921]
### Steps to reproduce

1. create a text file
2. edit on iOS and place cursor at the middle of text
3. press enter

### Expected behaviour

A new line is inserted at the cursor position.

### Actual behaviour

A new line is inserted at the begining of text...

**Comment by thisIsTheFoxe** [comment]
This seems like a duplicate of #1631 / nextcloud/text#1927
Can you confirm that the same is happening when opening the text editor with Safari on iOS?

**Comment by gabriel-joy** [comment]
No, it's not the same, in fact I can edit the file without any issues, but not enter :)

However, I have more info:

**The issue is reproducing only if the line terminator is LF,** that means for files created on iOS or with Notepad the issue does not reproduce.

I've made a copy paste from browser to a text file, that's how I ended up here :)

Please consider as low prio.

**Comment by thisIsTheFoxe** [comment]
Thanks for the additional info, we'll keep that in mind.
But I'm still unsure if this is an issue of the iOS app. Could you please try on your device using Safari? If the same thing happens there as well please transfer the issue over to [nextcloud/text](https://github.com/nextcloud/text).

Also, if possible include example files were it works and where it doesn't. 
Thanks!

**Comment by gabriel-joy** [comment]
You're right, the issue reproduce also in Safari but not in Edge for example.

**Comment by gabriel-joy** [comment]
I don't know how to transfer this issue.. Maybe because is closed?

**Comment by thisIsTheFoxe** [comment]
That's ok thanks for letting us know!
No problem, I'll move it. And the WebView we use in the iOS app uses Apple's WebKit (like Safari) so it's possible that other browser implementations handle things a little differently... :)
